### ISSUE TITLE
why calculate_ma_dist without forwarding sep_attention_dist in softmax before?

### ISSUE THREAD

**Issue by HaRry-qaq** [Jeryi-Sun/ReDEeP-ICLR#13]
in calculating JS divs:
`softmax_anchor_layer = F.softmax(sep_attention_dist, dim=-1) `

while in calculating ma dist:
`
sep_vocabulary_dist = F.softmax(sep_vocabulary_dist, dim=-1)
dist_diff = sep_vocabulary_dist - sep_attention_dist
`
